R/save_arcadia_plot.R

```diff
@@ -1,57 +1,93 @@
 #' Save Arcadia Plot
 #'
 #' @param plot ggplot object to be saved
-#' @param filepath The path where the plot will be saved
-#' @param panel_size The size of the panel, one of "full_wide", "float_wide", "half_square", "full_square", "float_square"
+#' @param filepath The path where the plot will be saved without a file extension
+#' @param panel_size The size of the panel, one of "full_wide", "float_wide", "half_square", "full_square", "float_square". Default is "full_square"
 #' @param transparent_bg Logical, whether to set the plot background to be transparent
+#' @param formats Vector of file formats to export the plot as. Default is "pdf". Can be "pdf", "png", or both.
 #'
 #' @return None
 #' @export
 #'
 #' @examples
 #' \dontrun{
 #' p <- ggplot(mtcars, aes(x = hp, y = mpg)) + geom_point()
-#' save_arcadia_plot("plot_full_wide.pdf", p, "full_wide")
-#' save_arcadia_plot("plot_float_wide.pdf", p, "float_wide")
-#' save_arcadia_plot("plot_half_square.pdf", p, "half_square"
-#' save_arcadia_plot("plot_full_square.pdf", p, "full_square")
-#' save_arcadia_plot("plot_float_square.pdf", p, "float_square")
+#' save_arcadia_plot("plot_full_wide", p, "full_wide")
+#' save_arcadia_plot("/path/to/figs/folder/plot_float_wide", p, "float_wide", formats = c("pdf", "png"))
+#' save_arcadia_plot("plot_half_square", p, "half_square", formats = "png")
+#' save_arcadia_plot("plot_full_square", p, "full_square")
+#' save_arcadia_plot("plot_float_square", p, "float_square")
```

Comment:
can you name all of your arguments here? there are enough now where it is hard to follow

Reply:
addressed